insert-cover-pages-example/index.ts

```diff
@@ -0,0 +1,152 @@
+import { Parameters } from './parameters';
+import * as parametersDescriptions from './parametersDescriptions';
+import { Validator } from './validator';
+import { MetaContext, createMetaContext } from './Utils/metaContextUtils';
+import { pathCombine } from './Utils/pathUtils';
+import {
+    Command,
+    Commands,
+    createCopyInputPageCommand,
+    createGroupBeginCommmand,
+    createPageSizeCommmand,
+    getCommandFileName,
+} from './Utils/commandUtils';
+
+export function getDescription(): ScriptDescription {
+    return {
+        displayName: 'Insert Cover Pages',
+        description:
+            'Takes a fixed document and adds it to the beginning or end of each document in a record',
+        category: 'Impositioning',
+        input: [
+            parametersDescriptions.InputParamInputFilePath,
+            parametersDescriptions.InputParamMetadataFilePath,
+            parametersDescriptions.InputParamPrefixPagePath,
+            parametersDescriptions.InputParamSufixPagePath,
+            parametersDescriptions.InputParamOutputFilePath,
+            parametersDescriptions.InputParamOutputType,
+            parametersDescriptions.InputParamProductionConfiguration,
+            parametersDescriptions.InputParamWorkingDirectory,
+        ] as (
```

Comment:
This `as` is very strange it is really needed?

Reply:
this file was commited by mistake, will be removed